packages/governance/src/views/proposal/components/instructionInput/mangoAddPerpMarketForm.tsx

```diff
@@ -0,0 +1,301 @@
+      groupConfig.mangoProgramId,
+      mangoGroup.publicKey,
+      makePerpMarketAccountInstruction.account.publicKey,
+      makeEventQueueAccountInstruction.account.publicKey,
+      makeBidAccountInstruction.account.publicKey,
+      makeAskAccountInstruction.account.publicKey,
+      mngoVaultAccount.publicKey,
+      governance.pubkey,
+      new BN(marketIndex),
+      I80F48.fromNumber(maintLeverage),
+      I80F48.fromNumber(initLeverage),
+      I80F48.fromNumber(liquidationFee),
+      I80F48.fromNumber(makerFee),
+      I80F48.fromNumber(takerFee),
+      new BN(baseLotSize),
+      new BN(quoteLotSize),
+      I80F48.fromNumber(rate),
+      I80F48.fromNumber(maxDepthBps),
+      new BN(targetPeriodLength),
+      new BN(mngoPerPeriod),
+    );
+
+    onCreateInstruction(instruction);
+  };
+
+  return (
+    <Form {...formDefaults} form={form} onFinish={onCreate}>
+      <Form.Item label="program id">
+        <ExplorerLink
+          address={governance.info.governedAccount}
+          type="address"
+        />
+      </Form.Item>
+
+      <AccountFormItem
+        name="mangoGroupId"
+        label="mango group"
+        required
+      ></AccountFormItem>
+
+      <Form.Item name="marketIndex" label="market index" required>
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item
+        name="maintLeverage"
+        label="maintenance position leverage"
+        initialValue={10}
+        required
+      >
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item
+        name="initLeverage"
+        label="initial position leverage"
+        initialValue={5}
+        required
+      >
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item
+        name="liquidationFee"
+        label="liquidation fee"
+        initialValue={0.05}
+        required
+      >
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item name="makerFee" label="maker fee" initialValue={0.05} required>
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item name="takerFee" label="taker fee" initialValue={0.05} required>
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item
+        name="baseLotSize"
+        label="base lot size"
+        initialValue={5}
+        required
+      >
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item
+        name="quoteLotSize"
+        label="quote lot size"
+        initialValue={5}
+        required
+      >
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item name="rate" label="rate" initialValue={0.05} required>
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item
+        name="maxDepthBps"
+        label="max depth bps"
+        initialValue={0.05}
+        required
+      >
+        <Input type="number" />
+      </Form.Item>
+
+      <Form.Item
+        name="maxNumEvents"
+        label="max number of events"
+        initialValue={1}
```

Comment:
some of these defaults are far away from a reasonable setting, please use the same defaults that the cli uses:
https://github.com/blockworks-foundation/mango-client-v3/blob/main/src/cli.ts#L264